crd/multitenancy/client.go

```diff
@@ -190,3 +191,28 @@ func (i *Installer) InstallOrUpdatePodNetworkInstance(ctx context.Context) (*v1.
 	}
 	return current, nil
 }
+
+type NodeInfoClient struct {
```

Comment:
Looks like we have implemented an `Installer` struct which seems abstracted enough to be used for NodeInfo CRD as well, is there a reason why we can't build upon that?

Reply:
`Installer` is for the CustomResource**Definition**, not for the instance of the CustomResource

Reply:
ah i see what you mean, not familiar with CNS creating the actual CR itself 🙂